linux/meshd-nl80211.c

```diff
@@ -556,7 +534,9 @@ static int add_unauthenticated_sta(
   ret = nl_send_auto_complete(nlcfg->cmd_sock, msg);
   sae_debug(
       MESHD_DEBUG,
-      "new unauthed sta (seq num=%d)\n",
+      "new unauthed %s sta (seq num=%d)\n",
+      ((elems->vht_cap) ? "VHT" :
+       (elems->ht_cap) ? "HT" : "NONHT"),
```

Comment:
Consider `NOHT` for consistency with other parts of the code, unless this is actually more consistent with something in the standard.

Reply:
I was indeed looking at something in the standard that said NONHT at this point but since this isn't  relevant to that I will change.